internal/operators/cnv/manifest.go

```diff
@@ -224,3 +225,15 @@ reclaimPolicy: Delete
 volumeBindingMode: Immediate
 parameters:
   storagePool: sno`
+
+const schedstatsMachineConfigManifest = `
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
```

Comment:
After further discussion (I put a link in the ticket), it looks like we also want a MachineConfig for the control plane nodes as well, so that we'll do the right thing if the user turns on `schedulableMasters` (either on day 1 or day 2).